apps/next/src/pages/LockScreen/components/ForgotPassword.tsx

```diff
@@ -16,7 +16,7 @@ import { ResetExtensionStateHandler } from '~/services/storage/handlers/resetExt
 
 import { WarningMessage } from '@/components/WarningMessage';
 import { PageTopBar } from '@/components/PageTopBar';
-import { dialogSlots } from '@/pages/AccountManagement';
+import { dialogSlots } from '@/pages/AccountManagement/components/Dialog';
```

Comment:
It's still cross-referencing to `AccountManagement` from a different page. It needs to go higher.

Since we want `<Dialog />` to have the same default `slots` and `slotProps`, we should create `@/components/Dialog` that pre-defines them, and use this new `<Dialog />` with default values in both `AccountManagement` and `in `ForgotPassword`.

Oor, place those defaults props in `@/config/dialog` and reference it from both places.